.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
     - os: linux
       env: NODE_VERSION=10
     - os: linux
-      env: NODE_VERSION=11
+      env: NODE_VERSION=12
```

Comment:
This isn't "changing" the Node version. You've removed Node version 11 from the build and added 12. 

You should add a new matrix entry

Reply:
It's generally practice to add only the stable branch in testing once they are released hence removal of 11 and addition of 12.

Reply:
That’s fair, 9 isn’t tested either but I don’t think it would be a bad thing if 9 and 11 were tested